src/server/routes/monthData.js

```diff
@@ -54,14 +99,35 @@ function selectClientMonthData(req, res) {
       console.error(`Status Code: ${error.statusCode}, Error Message: ${error.code}`);
 
       res.json({
-        statusCode: error.statusCode,
+        statusCode: error.statusCode || 500,
         errorMessage: error.code,
       });
     });
 }
 
+/**
+ * selectClientMonthDataPost(req, res, next)
+ * Handles the requests from the form submit button (when no JS).
+ * It redirects to the route to execute the function for server side requesting.
+ */
+function selectClientMonthDataPost(req, res) {
+  const season = (req.body.season) ? `${req.body.season}-01` : '';
+  const audience = req.body.audience;
+
+  if (!season || !audience) {
+    console.log(
```

Comment:
You should also redirect to the 404 page on the server side if the from submission doesn't work.

Reply:
Will do it on the next PR.

Reply:
Do it for this one to make it complete. It isn't hard to do.

Reply:
I hear you. It is not because hard or easy. I have the plan to do error handlings in another ticket so I can track it easier.